Add missing packages to requirements.txt

damageboy opened the pull request:
Missing pacages...

damageboy committed:
Add missing packages to requirements.txt, remove publish per discussion

damageboy committed:
somewhat better install instructions

JonathanVusich reviewed (changes requested):
It would probably actually be best to remove the `publish` method in `main.py`. That was just something that I was experimenting with, and wasn't really intended to make it into production. In reality, updates to the database should be done through a PR and not just pushing directly to master. That way we can avoid the `sh` requirement as well. Would you mind making these changes? Otherwise it's looking good!

Edit: I have added CI and tests as well to this repo. Perhaps you should pull those changes and integrate them as well.

damageboy commented:
Sure thing...

damageboy commented:
I think I've addressed your request, but I can't mark anything as resolved since I rebased the whole thing on top of master...

Let me know if you're good with it.

I'd love to rebase my other thing (with ecc + parallelism support) and submit a pull request for that as well, I'd just want to get this silly thing out of the way...
